master/static/srv/get_checkpoint.sql

```diff
@@ -1,7 +1,8 @@
 SELECT
-    c.uuid AS uuid,
+    c.uuid::text AS uuid,
```

Comment:
blocking: will this mess with the original checkpoint endpoints?

Reply:
I've tested all of the endpoints. I need to add `state` to `ExportableCheckpoint` but otherwise they work.